charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-apiserver.rules.yaml

```diff
@@ -85,4 +85,10 @@ groups:
         summary: 'The API server has too many failed attempts to log audit events'
 
   - record: shoot:kube_apiserver:sum_by_pod
-    expr: sum(up{job="kube-apiserver"}) by (pod)
\ No newline at end of file
+    expr: sum(up{job="kube-apiserver"}) by (pod)
+
+  - record: shoot:container_network_transmit_bytes_total_apiserver:sum
+    expr: sum(rate(container_network_transmit_bytes_total{pod=~"kube-apiserver(.+)"}[10m]))
```

Comment:
this will present them as bytes right, does it makes sense to represent them as MB?